liveview-android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest package="org.phoenixframework.liveview" xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools">
+
+    <application
+
+        android:usesCleartextTraffic="true" />
+
```

Comment:
Any idea why the automatic generator leaves so many blank lines?

Reply:
this was actually because i removed a few things that left empty spaces, removed unnecessary line breaks